editoast/src/error.rs

```diff
@@ -170,3 +170,34 @@ impl EditoastError for reqwest::Error {
         "editoast:ReqwestError"
     }
 }
+
+#[derive(Debug)]
+pub enum EditoastUrlError {
```

Comment:
Should not be in this module. You can move it where we're using it. You will probably need to split it.

Reply:
Done.